tests/testthat/test-utils_data_handling.R

```diff
@@ -29,13 +29,14 @@ test_that("quantile_to_interval_dataframe() works", {
   quantile[c(1, 3, 11, 13), c("observed", "predicted", "quantile_level") := NA]
   # in this instance, a problem appears because there is an NA value both
   # for the upper and lower bound.
-  expect_message(
+  
+  expect_warning(
```

Comment:
The tests are currently failing - it seems like this is still producing a message at the moment, not a warning..

Reply:
Good point. Argh, the code is now a bit messy in order to flexibly allow message/warning depending on what's actually produced.

OTOH, we could avoid this by doing instead:

```r
skip_if_not_installed("data.table", "1.16.0")
expect_warning(..., "<v1.16.0 message only>")
```

Then the test will stop running until data.table is upgraded.